addons/towing/XEH_postInit.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+["MouseButtonDown", LINKFUNC(onMouseButtonDown)] call CBA_fnc_addDisplayHandler;
+["MouseButtonUp", LINKFUNC(onMouseButtonUp)] call CBA_fnc_addDisplayHandler;
+GVAR(mouseLeft) = false;
+GVAR(mouseRight) = false;
+
+GVAR(cancel) = false;
+GVAR(canAttach) = false;
+
+[QGVAR(lockVehicle), {
```

Comment:
there are already events for it https://github.com/acemod/ACE3/blob/ee4431980146091fa816b7b9da87dde6e3ca3094/addons/common/XEH_postInit.sqf#L127-L134